docs/installation/services.rst

```diff
@@ -30,7 +30,7 @@ The login page is loaded, and should resemble the following:
 
 .. image:: img/ui/login.png
 
-Log in with the default login / password (admin / admin).
+Log in with the default login / password (admin@metalk8s.invalid / password).
```

Comment:
We use monospace font to express content the user needs to type in, so:

```suggestion
Log in with the default login / password (``admin@metalk8s.invalid`` / ``password``).
```

Reply:
Thanks! 
But just let you know I removed the change for login/password, since it's already in Claude's PR. My PR only includes the screenshot update for Dex.